lib/chewy/type/adapter/orm.rb

```diff
@@ -110,14 +110,15 @@ def load(ids, **options)
       private
 
         def import_objects(collection, options)
-          hash = Hash[identify(collection).zip(collection)]
+          collection_ids = identify(collection)
+          hash = Hash[collection_ids.map(&:to_s).zip(collection)]
```

Comment:
Ids exist in different formats, like mongo id or string or integer, string is a universal representation for all of them, I'm using it a lot.